src/avram/criteria_extensions/includes_criteria.cr

```diff
@@ -0,0 +1,27 @@
+module Avram::IncludesCriteria(T, V)
+  macro included
+    # WHERE `value` = ANY(column)
+    def includes(value) : T
```

Comment:
I also debated on `contains`, but with `includes`, we could also add an `excludes` method... What would be the opposite of `contains`, `omits`? 🤷‍♂️

Reply:
The Avram DSL here is close enough to sql that I think includes makes sense.

Reply:
Actually, I misunderstood the purpose of this when I said that. But I still think that includes is as good a name as any. Sorry for the email spam from my github comments 🤪

Reply:
😂  cool. Second confirmation is what I needed. Thanks